gce/run-e2e.sh

```diff
@@ -15,11 +15,13 @@ sleep ${INIT_TIMEOUT:-1s}
 # Taint the Linux nodes to prevent the test workloads from landing on them.
 # TODO: remove this once the issue is resolved:
 # https://github.com/kubernetes/kubernetes/issues/69892
-LINUX_NODES=$(kubectl get nodes -l beta.kubernetes.io/os=linux -o name)
-LINUX_NODE_COUNT=$(echo ${LINUX_NODES} | wc -w)
-for node in $LINUX_NODES; do
-  kubectl taint node $node node-under-test=false:NoSchedule
-done
+if [[ ! -v NO_LINUX_POOL_TAINT ]]; then
```

Comment:
The double negative here (! NO) makes this harder to understand - is it possible to make this "if LINUX_POOL_TAINT" ?

Reply:
it's still need for <1.23. I discussed with them about that. so going forward won't need it. regarding the double negative, I still want to keep the default to taint - not sure what all jobs utilizing this as test cmd, hence opting for having it to be disabled if NO_LINUX_POOL_TAINT is set.